resources/views/inc/sidebar.blade.php

```diff
@@ -73,7 +73,7 @@
                 @if(Auth::user()->isAbleTo('roster'))
                     <a class="nav-link {{ Nav::urlDoesContain('dashboard/admin/bronze-mic') }}" href="/dashboard/admin/bronze-mic">Award Management</a>
                 @endif
-                @if(Auth::user()->isAbleTo('staff'))
+                @if(Auth::user()->isAbleTo('staff') || Auth::user()->hasRole('events-team'))
```

Comment:
As Ian said, this won't work because the events team doesn't have access to the administration menu. Please add a button on the events view (events team does have access to that) which links to the realops admin page (button should only be visible when realops toggle is on).